### DISCUSSION TITLE
How should we redefine/overlap the css vars?

### DISCUSSION THREAD

**Discussion by fablexis** [vueform/vueform#165] [Questions]
I'm taking a look at the docs, and based on [what it says ](https://vueform.com/docs/styles-and-layout#global-customization-1), if we want to redefine the values of any property in the CSS, we can declare a global CSS like this:

```
:root, :before, :after, * {
  --vf-border-color-input: #14b8a6;
  --vf-bg-input: #ccfbf1;
}
```

I'm creating a CSS file for that and just put it in my `main.ts` file to have it globally, but this seems to be not working as the prior thing, when I inspect the styles, I see that the values that are taken into account are the ones that belong to the theme instead of the CSS vars that I defined in my global scope with my CSS file.

Here is the screenshot:

![image](https://github.com/vueform/vueform/assets/7627658/ee410697-ad1f-454b-b123-7bfa5aa02a60)

The first values are the ones that belong to the theme, the others are the ones that I defined in my CSS file....

So, how should I define the CSS vars to overlap the theme's values? where should I put the import of that CSS file since the priority seems to be assigned to the ones that belong to the theme? Am I missing something from the docs?

@adamberecz

**Comment by adamberecz** [comment]
You should somehow make sure that the ones defined by you are loaded later.

An alternative is to make the scope narrower, eg:

```vue
<template>
  <Vueform add-class="my-form" >
</template>

<style>
.my-form, .my-form *, .my-form :before, .my-form :after {
  --vf-border-color-input: #14b8a6;
  --vf-bg-input: #ccfbf1;
}
</style>
```

You can also add a class to all forms globally:

```js
// vueform.config.js

export default {
  addClasses: {
    Vueform: {
      form: 'my-form',
    },
  },
}
